ublox_gps/src/node.cpp

```diff
@@ -1813,10 +1813,15 @@ void rtcmCallback(const rtcm_msgs::Message::ConstPtr &msg) {
   gps.sendRtcm(msg->message);
 }
 
+void spartnCallback(const std_msgs::UInt8MultiArray::ConstPtr &msg) {
+  gps.sendSpartn(msg->data);
+}
+
 int main(int argc, char** argv) {
   ros::init(argc, argv, "ublox_gps");
   nh.reset(new ros::NodeHandle("~"));
   ros::Subscriber subRtcm = nh->subscribe("/rtcm", 10, rtcmCallback);
+  ros::Subscriber subRtcm = nh->subscribe("/spartn", 10, spartnCallback);
```

Comment:
This is overriding the ROS subscriber for `subRtcm`

Reply:
Fixed